src/components/ebay-calendar/calendar.stories.js

```diff
@@ -0,0 +1,150 @@
+            control: { type: 'boolean' },
+            description:
+                'Date cells are contained in buttons for interactive calendars, and spans otherwise',
+            table: {
+                defaultValue: {
+                    summary: 'false',
+                },
+            },
+        },
+        numMonths: {
+            type: 'number',
+            control: { type: 'number' },
+            description: 'Number of months to be displayed at once',
+            table: {
+                defaultValue: {
+                    summary: 'false',
+                },
+            },
+        },
+        range: {
+            type: 'boolean',
+            control: { type: 'boolean' },
+            description: 'True if selecting a range, false if a single value',
+            table: {
+                defaultValue: {
+                    summary: 'false',
+                },
+            },
+        },
+        locale: {
+            type: 'text',
+            control: { type: 'text' },
+            description: 'Locale of the date picker',
+            table: {
+                defaultValue: {
+                    summary: 'navigator.language',
+                },
+            },
+        },
+        disableBefore: {
+            type: 'date',
+            control: { type: 'date' },
+            description: 'First date that may be selected',
+            table: {
+                defaultValue: {
+                    summary: 'undefined',
+                },
+            },
+        },
+        disableAfter: {
+            type: 'date',
+            control: { type: 'date' },
+            description: 'Last date that may be selected',
+            table: {
+                defaultValue: {
+                    summary: 'undefined',
+                },
+            },
+        },
+        disableWeekdays: {
+            type: 'array',
+            control: { type: 'array' },
+            description:
+                'List of weekdays that are disabled. Must be an array of numbers, where Sunday is `0` and Saturday is `6`',
+            table: {
+                defaultValue: {
+                    summary: 'undefined',
+                },
+            },
+        },
+        disableList: {
+            type: 'array',
+            control: { type: 'array' },
+            description:
+                'List of specific days that are disabled. Should be a list of date objects, but also accepts timestamps or ISO strings',
+            table: {
+                defaultValue: {
+                    summary: 'undefined',
+                },
+            },
+        },
+        onSelect: {
+            action: 'on-select',
+            description: 'Triggered when a date is selected',
+            table: {
+                category: 'Events',
+                defaultValue: {
+                    summary: '{ iso }',
+                },
+            },
+        },
+        onMonth: {
+            action: 'on-month',
```

Comment:
I would make this more explicit
On-month-change or on-month-update etc

Reply:
I would like to change this, but I am unable to get storybook working with multi-word events. I don't this works for any of our components.